src/streaming/MediaPlayer.js

```diff
@@ -638,6 +638,16 @@ function MediaPlayer() {
     }
 
     /**
+     * @memberof module:MediaPlayer
+     * @see {@link module:MediaPlayer#setLiveDelay setLiveDelay()}
+     * @instance
+     * @returns {number} Current live stream delay in seconds
+     */
+    function getLiveDelay() {
+        return mediaPlayerModel.getLiveDelay();
```

Comment:
Good API to have as we are still missing live delay subtraction from DVR Metric.  Meaning that control bar will render duration without live delay factored.  I am fixing this now but this is useful.  Thanks.